src/AppBundle/v2/DTO/ClientDto.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace AppBundle\v2\DTO;
+
+class ClientDto implements \JsonSerializable
+{
+    private $id;
+    private $caseNumber;
+    private $firstName;
+    private $lastName;
+    private $email;
+    private $reportCount;
+    private $ndrId;
+
+    /**
+     * @param $id
+     * @param $caseNumber
+     * @param $firstName
+     * @param $lastName
+     * @param $email
+     * @param $reportCount
+     * @param $ndrId
+     */
+    public function __construct($id, $caseNumber, $firstName, $lastName, $email, $reportCount, $ndrId)
```

Comment:
would it be better to have the last two parameters optional?

Reply:
Maybe, but not sure. Leave it for now?